src/OrleansRuntime/MembershipService/MembershipOracleData.cs

```diff
@@ -24,6 +26,12 @@ internal class MembershipOracleData
         internal readonly string MyHostname;
```

Comment:
should clusterSizeStatistic and clusterStatistic fields be used now? Because they are currently initialized but not ever used.